p4-16/spec/grammar.mdk

```diff
@@ -253,6 +254,7 @@ functionPrototype
 methodPrototype
     : optAnnotations functionPrototype ';'
     | optAnnotations TYPE_IDENTIFIER '(' parameterList ')' ';'
+    | optAnnotations ABSTRACT functionPrototype ';'
```

Comment:
this is clearly experimental, part of the abstract methods.

Reply:
removed